CMakeLists.txt

```diff
@@ -9,5 +9,7 @@
 
 cmake_minimum_required( VERSION 3.3.2 FATAL_ERROR )
 
-project( ioda_data VERSION 2.7.0 DESCRIPTION "IODA Test Files" )
+project( ioda-data VERSION 2.7.0 DESCRIPTION "IODA Test Files" )
```

Comment:
I think we should not use the hyphen "-" in the project name. The reason is that you cannot use a hyphen in a shell environment variable name. Ie the recommendation about setting "ioda-data_DIR" variable is not possible to do:
```
CMake Error at ioda/CMakeLists.txt:76 (find_package):
  By not providing "Findioda-data.cmake" in CMAKE_MODULE_PATH this project
  has asked CMake to find a package configuration file provided by
  "ioda-data", but CMake did not find one.

  Could not find a package configuration file provided by "ioda-data" with
  any of the following names:

    ioda-dataConfig.cmake
    ioda-data-config.cmake

  Add the installation prefix of "ioda-data" to CMAKE_PREFIX_PATH or set
  "ioda-data_DIR" to a directory containing one of the above files.  If
  "ioda-data" provides a separate development package or SDK, be sure it has
  been installed.
```